cmd/dlv/cmds/commands.go

```diff
@@ -78,6 +81,7 @@ func New() *cobra.Command {
 	RootCommand.PersistentFlags().BoolVarP(&Log, "log", "", false, "Enable debugging server logging.")
 	RootCommand.PersistentFlags().BoolVarP(&Headless, "headless", "", false, "Run debug server only, in headless mode.")
 	RootCommand.PersistentFlags().BoolVarP(&AcceptMulti, "accept-multiclient", "", false, "Allows a headless server to accept multiple client connections. Note that the server API is not reentrant and clients will have to coordinate")
+	RootCommand.PersistentFlags().IntVar(&ApiVersion, "api", 1, "Selects API version when headless")
```

Comment:
nit: Let's change this to `api-version`, and specify in the help output what version numbers are accepted.